task/progress.go

```diff
@@ -16,7 +16,9 @@ var progressReportBuckets = []float64{0, 0.25, 0.5, 0.75, 1}
 const minProgressReportInterval = 10 * time.Second
 const progressCheckInterval = 1 * time.Second
 
-func ReportProgress(ctx context.Context, lapi *livepeerAPI.Client, taskID string, size uint64, getCount func() uint64) {
+func ReportProgress(ctx context.Context, lapi *livepeerAPI.Client, taskID string, size uint64, getCount func() uint64, startPercentage int, endPercentage int) {
+	startPercentage = int(math.Max(float64(startPercentage), 0)) // >= 0, <= 100, smaller than endPercentage
+	endPercentage = int(math.Min(float64(endPercentage), 100))   // >= 0, <= 100, larger than startPercentage
```

Comment:
**nit:** I think the comments are unnecessary since the code is relatively simple/self-explanatory (they're also innacurate since we're only enforcing 1 of the 3 things it mentions)